matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/SyncTask.kt

```diff
@@ -239,6 +255,45 @@ internal class DefaultSyncTask @Inject constructor(
         }
     }
 
+    /**
+     * Aggregator to send stat event.
+     */
+    class SyncStatisticsData(
+            val isInitSync: Boolean,
+            val isAfterPause: Boolean
+    ) {
+        val startTime = SystemClock.elapsedRealtime()
+        var requestInitSyncTime = startTime
```

Comment:
not a blocker, could be nice to avoid mutating the `SyncStatisticsData` and instead create a concrete `StatisticEvent.*` for the different types of sync (with immutable timestamps), I can understand why it's done this way as we're accumulating the data